plugins/modules/packaging/language/cpanm.py

```diff
@@ -71,10 +71,14 @@
 '''
 
 EXAMPLES = '''
-# install Dancer perl package
+# install latest version of Dancer perl package
 - cpanm:
     name: Dancer
 
+# install Dancer 1.3513 or later Perl package
+- cpanm:
+    name: Dancer~1.3513
```

Comment:
Shouldn't it be:
```suggestion
    name: Dancer
    version: ~1.3513
```
?

Reply:
Also, does `name: Dancer~1.3513` or `version: ~1.3513` actually work with `_is_package_installed()`?